src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -600,13 +600,15 @@ Copyright (c) .NET Foundation. All rights reserved.
   ============================================================
   -->
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_IsExecutable)' == 'true'">
-    <ProjectCapability Include="CrossPlatformExecutable" />
-  </ItemGroup>
-
-  <!-- Reference Manager capabilities -->
+  <!-- NETCoreApp capabilities -->
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <ProjectCapability Remove="ReferenceManagerAssemblies" />
```

Comment:
Existing: What does ReferenceManagerAssemblies do and why weren't we removing it for .NETStandard?

Reply:
These properties control the available tabs in the reference manager that is opened from the dependencies node context menu -> "Add Reference"
<img width="590" alt="refmang" src="https://user-images.githubusercontent.com/36282608/53598528-def16180-3b59-11e9-8802-0c93d392974b.PNG">

Reply:
Thanks. Looks like it's it just a bug that we're hiding this for .NET Core and not .NET Standard...

Reply:
I'm not sure - @Pilchie @AaronRobinsonMSFT  should we remove both assemblies and COM for .NET Standard?

Reply:
We can track this separately: https://github.com/dotnet/sdk/issues/2996

I just spotted it while reviewing your change, but it's existing so no need to come to a conclusion on this change.

Reply:
>should we remove both assemblies and COM for .NET Standard?

@etbyrd I can only speak for COM and .NET Standard. Making the call for regular assemblies is above my pay grade.

@jeffschwMSFT can you provide some guidance on who could make the above decision?

Reply:
Yes we should remove this page for .NET Standard, however, making this change here will only take effect for the new SDK. We should move this into design-time targets on our side.